### DISCUSSION TITLE
Can location refresh be changed or smoothed from 1 second interval

### DISCUSSION THREAD

**Discussion by mirror176** [osmandapp/OsmAnd#13611] [Maps Related]
While zoomed in on the map and in a vehicle, the map seems to do a smooth scroll+zoom every second as the vehicle position+direction is updated. Is there a way to have it update position faster or animate the vehicle based on linear+angular velocity so it could seem to move/turn more realistically? If not, is there a way to disable the smooth scroll+zoom and just let it jump to a new map position+angle as it gets new vehicle data when following vehicle view?

**Comment by vshcherb** [comment]
Sounds interesting to create a more like cinematic effect but I wouldn't call it realistic cause the camera movement up/down doesn't correlate with car movement in general
